### ISSUE TITLE
Watch Recordings: Titles Appear OK With Coverart, but Empty Tile Also Appears

### ISSUE THREAD

**Issue by billmeek** [MythTV-Clients/MythtvPlayerForAndroid#66]
Just documenting this for now because I don't know how
to recreate it. But it's been happening consistantly.

Symptom: A 'normal' tile appears in Watch Recordings,
that is it has coverart and when tapped, all of the episodes
are displayed.

The issue is that at times, there is a 2nd tile with the same
title, but no coverart and when tapped, nothing is displayed.

SQL from the BE and app:
``` sql
mysql> SELECT originalairdate,subtitle,season,episode,inetref,recordedid FROM recorded WHERE title='Ask This Old House' ORDER BY originalairdate;
+-----------------+-------------------------------------------------------------------------------------+--------+---------+----------------+------------+
| originalairdate | subtitle                                                                            | season | episode | inetref        | recordedid |
+-----------------+-------------------------------------------------------------------------------------+--------+---------+----------------+------------+
| 2015-04-18      | Lightning Protection; Reverse Osmosis Water Filter Install                          |     13 |      20 | ttvdb.py_74326 |       2904 |
| 2015-04-25      | Exterior Lighting Install; Main Water Shutoff Replacement                           |      0 |       0 | ttvdb.py_74326 |       3006 |
| 2015-05-02      | Mailbox Replacement; Ask Kevin: Window Anatomy; Re-use an Old Door in a New Opening |      0 |       0 | ttvdb.py_74326 |       3061 |
| 2015-05-09      | Noisy Baseboard Heating Fix; Recessed Kitchen Lighting Install                      |      0 |       0 | ttvdb.py_74326 |       3127 |
| 2015-05-16      | Maine Tropical Houseplants; USA Flag Build                                          |     13 |      24 | ttvdb.py_74326 |       3175 |
+-----------------+-------------------------------------------------------------------------------------+--------+---------+----------------+------------+
5 rows in set (0.00 sec)

sqlite> SELECT program_air_date,program_sub_title,program_season,program_episode,program_inetref FROM programs WHERE program_title='Ask This Old House' ORDER BY program_air_date;
1429333200000   Lightning Protection; Reverse Osmosis Water Filter Install                              13    20    ttvdb.py_74326 
1429938000000   Exterior Lighting Install; Main Water Shutoff Replacement                               0     0     ttvdb.py_74326 
1430542800000   Mailbox Replacement; Ask Kevin: Window Anatomy; Re-use an Old Door in a New Opening     0     0     ttvdb.py_74326 
1431147600000   Noisy Baseboard Heating Fix; Recessed Kitchen Lighting Install                          0     0     ttvdb.py_74326 
1431752400000   Maine Tropical Houseplants; USA Flag Build                                              13    24    ttvdb.py_74326
```

**Comment by billmeek** [comment]
It appears that to duplicate this 'phantom tile',  just  leave
the tablet on while the BE is also on and a new recording is
made. Perhaps it's that during the recording, the inetref
isn't available.

Example (Chicago PD below with blank inetref):
![screenshot_2015-05-21-11-14-23](https://cloud.githubusercontent.com/assets/1831699/7753382/abeac66c-ffaa-11e4-8c5c-4ac29cb91a33.jpg)

The fix is to stop the app and delete cache and data.
Swiping down doesn't refresh the app's DB with the
inetref.

**Comment by billmeek** [comment]
Yet another accidental test finding. The 'phantom tile' vanishes
if I pick up the tablet and rotate it from landscape to portrait.
Happened on 4.0.4 and 5.1.

**Comment by billmeek** [comment]
Also note that I'm running with the MythTV Services API fix that
allows titles without inetrefs to be returned. That's going to hide
the issue from anyone without the patch.

The ticket was accepted 2 weeks ago by stuartm and written against
0.27. Patches for 0.27 and master were submitted, so I'm hopeful that
they will get released. https://code.mythtv.org/trac/ticket/12446

**Comment by dmfrey** [comment]
@billmeek does this still occur with the fix to issue #49?

Or is this the case where something is currently recording?  It is showing
the title, but no recordings appear because there status is not one we are
listing (i.e. recording, livetv, etc.)?

**Comment by billmeek** [comment]
It appears to happen when something is recording and the app
retrieves the list of shows.

Now, with 49, the phantom is self-healing. If I press on the title with no
coverart, nothing is displayed (same as before) but if I then
press on the title with coverart, it is AND the phantom goes
away when returning from the episode list to Watch Recordings.
